web/yo/app/views/gene.html

```diff
@@ -71,6 +72,7 @@ <h3 ng-if="reviewMode">Gene Type:
             <p ng-show="signatureCheck(gene.type_review)" style="float:right;font-size:15px;">Updated by {{gene.type_review.get('updatedBy')}} at {{gene.type_review.get('updateTime')}}</p>
             <i class="fa fa-check" ng-class="iconClass('accept', gene.type_review)" ng-if="iconExist('accept', gene.type_review)"  ng-click="accept(null, 'GENE_TYPE', null, null, null, null, gene.type_review)"></i>
```

Comment:
Can you combine these three icons into  a directive? You don't need to save the loading status into the document.